.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+Please explain the changes you have made here.
+
+## Jira Ticket
+
+Please add a link to the related Jira ticket (if applicable).
+
+## Checklist
+
+These are items (excluding GitHub Checks) which should be confirmed before a branch is ready to merge.
+
+- [ ] Acceptance Criteria met
```

Comment:
I'd remove this one, as it applies to the story, but not necessarily to a pull request. There can (and usually should) be multiple PRs per story. A single PR doesn't need to satisfy any acceptance criteria by itself.

Reply:
I'd say that ensuring the code meets the acceptance criteria/user requirements is essential for any code review. By "acceptance criteria", we just mean "Does this code solve the underlying problem/user requirement?"
This item in the checklist is just to trigger peoples' minds to check that, and if there's later PRs to meet more acceptance criteria, that's fine.

Reply:
> I'd say that ensuring the code meets the acceptance criteria/user requirements is essential for any code review.

It is generally considered the best practice to create many small pull requests rather than big ones. Those small PRs don't need to satisfy any acceptance criteria on their own (and usually don't), so we shouldn't expect it from them. It's good to have a checklist like this, but I think it would be better not to have this point. Or maybe we could add `if applicable` or sth?